### PR TITLE
added german translation

### PR DESCRIPTION
**Pull request by Frxhb**
german translation in informal german.

### CODE REVIEW

**Comment by Frxhb** [comment]
Et voilà
So whats up with Support translation for [piholeremote](https://github.com/jojost1/piholeremote-translations)? ;)

File: Support/support-de.json [hunk 1]
```diff
@@ -185,33 +185,33 @@
...
-                    "text": "First, make sure you are logged into iCloud on your iPhone. Then restart your phone & watch, open the iPhone app, then open the Watch app. If that doesn't work, a fresh reinstall of the app should work. Sorry for the inconvenience!"
+                    "title": "Warum bleibt die Apple Watch App an einem Ladekreis hängen?",
+                    "text": "Stelle zunächst sicher, dass Du bei iCloud auf Deinem iPhone angemeldet bist. Starte dann Dein Telefon und Deine Uhr neu, öffne die iPhone-App und dann die Watch-App. Wenn das nicht funktioniert, sollte eine Neuinstallation der App helfen. Ich entschuldige mich für die Unannehmlichkeiten!"
                 },
                 {
                     "id": 4,
-                    "title": "I'm using Home Assistant, how do I setup the app?",
-                    "text": "In Home Assistant > Configuration > Add-ons, Backups & Supervisor > AdGuard Home > Configuration, there is a field for 'Network Host'. Assign a port-number there for the web interface and enter it in the app."
+                    "title": "Ich verwende den Home Assistant. Wie richte ich die App ein?,
```
Comment (jojost1) [inline Support/support-de.json:208]: Small mistake here I think, values should always start and end with `"` and end with a `,` if another key-value pair comes behind it

**Review by jojost1 — COMMENTED** [review]
One small thing

**Comment by jojost1** [comment]
> So whats up with Support translation for [piholeremote](https://github.com/jojost1/piholeremote-translations)? ;)

Good one, I'll post these files as well very soon! 😄

File: Support/support-de.json (hunk 1 above)
Comment (jojost1) [reply Support/support-de.json:208]: I'll fix this and push the fix myself 😄 so I can make a new beta build now

**Review by jojost1 — APPROVED** [review]

File: Support/support-de.json (hunk 1 above)
Comment (Frxhb) [reply Support/support-de.json:208]: Oh I am sorry. I accidentally deleted a " ... Sorry !:D
Was ist just in this line?

Comment (jojost1) [reply Support/support-de.json:208]: Yes was just here 😄 but I fixed it with a separate commit

**Pull request merged**